webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -42,6 +42,7 @@ export class TasksComponent implements OnInit, OnDestroy {
   selectedTask;
   errorStack;
   hasTasks;
+  tasksReady;
```

Comment:
I don't understand why we need an additional flag (this.tasksReady) instead of using the loading flag (this.loading)

moving the loading just before the catch or in the finally block
```
      this.tasksActions.setTasksList(hydratedTasks);
      this.loading = false;

    } catch (exception) {
```

`<p class="loading-status" *ngIf="!errorStack && !loading && !hasTasks && !tasksDisabled">{{'task.list.complete' | translate}}</p>`